sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -522,6 +525,10 @@ export interface IndexingPolicy {
     excludedPaths?: IndexedPath[];
     includedPaths?: IndexedPath[];
     indexingMode?: keyof typeof IndexingMode;
+    // Warning: (ae-forgotten-export) The symbol "SpatialIndex" needs to be exported by the entry point index.d.ts
```

Comment:
@southpolesteve is this an real issue that needs attention?

Reply:
It is fine. I have missing exports elsewhere. 

The real solution here is the build should fail if there are missing exports. These shouldn't be warnings. But they all need to be fixed first.

Reply:
Yes, I just logged #7451

Reply:
oops you are talking about different thing.